### PR TITLE
[DOC] Improve High Availability Guide

### PR DESCRIPTION
**Pull request by yaooqinn**
### _Why are the changes needed?_

image

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @pan3793

File: docs/deployment/high_availability_guide.md [hunk 1]
```diff
@@ -15,20 +15,70 @@
...
+<div align=center>
+
+![](../imgs/kyuubi_logo.png)
+
+</div>
+
 # Kyuubi High Availability Guide
 
-As an enterprise-class ad-hoc SQL query service built on top of [Apache Spark](http://spark.apache.org/), Kyuubi takes high availability(HA) as a major characteristic, aiming to ensure an agreed level of service availability, such as a higher than normal period of uptime.
+As an enterprise-class ad-hoc SQL query service built on top of [Apache Spark](http://spark.apache.org/), Kyuubi takes high availability (HA) as a major characteristic, aiming to ensure an agreed level of service availability, such as a higher than normal period of uptime.
```
Comment (pan3793) [inline docs/deployment/high_availability_guide.md:26]: use `https` for Apache Spark link

File: docs/deployment/high_availability_guide.md [hunk 2]
```diff
@@ -15,20 +15,70 @@
...
...KN2+3cG3rvIm4HvLnEd83jc/r9ju4/jvE3A3kc5cGXrxNxsPMpFQc7PyfjYMejtTwl5E9NkuBaSPE7bOspQnhPtV9mgO84kjfW9+we3re+F40gDyhBvdCB4d32f6XdvTz8noj/rG7fvnztxO8neO3t67D6wIWaL0ovGCpJFPH9kdFzoPZLSi/MPdC6Rzboz8VLv7/4Qr9XZvh3Vfme2cq/o8r3QrUkCZ/+3lT//gE9Q+n7dNddzz5+kY/5exKi/9fTD1O49EotCPqvTz+g5BfTD1/myr872XqvKvzoSB7DmrsX+QvO6oc91fPXr5v+GKPf90xEfNvRfW8V4W7Dv8SH4feZ/CcFwz+8C/ubZlCbti6O4b2LDmHkdY8qfgzbIHlSjamhw+2UP+fA7rMMH8WBfTFrTqDoA4VjBEVjOI4wGIv/NWeGMdQDg+A0yRAkjVPMPeq5N0MiDwyOEDhCESSKUdTf6umov1BW+v4QLE6bW1b2YQP4n5ppRRCK+pGZ1mcb/hUuDCHRf1u4TyF/jw/7z3gkAsEfGIJip/iHolj2ddhNIH89viJY5gO7pF+YFT6WqfunrgG76CeL/c3O6j59/bf6qhe+CceffNNLH0Z9p6N6NtRfUpd4/u2/Jdai0HcUEPqpr/spknodTRMI+0CSKEIxJEPhOPuF//huL8W8Dsjwv3kRGvX1LPBfExu1Rfk7crtn9/JLcrtng/slgRH+Wms+flxE/tP+5oO4kS8XDxJ/dWr+yxNRxBfX8rsdB/ZOh/6g48C/4jhe16F+4kT3abuvlJd+iVvKw+jXBUEo/vMJG4Kw7A8kbL/ML91n7V4o5Yd3S9TbdXO/xS09V80fqFvn3CvnwJcj6DeL52BjFdbpdKtg9Pt2PerRe3zrhr8z+vpghXecZR9wnMQZlCBIYvrnizwR/Yk8EUUept+jFEmDPJR5nSbiJPvAkJOzRaZEkmWZL56C+t1O9u3ywh/2jV+px78JzX6m8v4bMsEv3dxfywy/Nzi7u4Ff4AQRknztBD9+cMa8p2b/Yi/4QZwWSRAPOEog97/XJQKKwB4Q9F7g/FGvRRL4A8HSOEOjKMKQX5QfSOr2qCeFEujksqZW/lavxXy93g691i/0Ws9m+1/zWp+e1k29OPzziil89n8=&lt;/diagram&gt;&lt;/mxfile&gt;&quot;}"></div>
```
Comment (pan3793) [inline docs/deployment/high_availability_guide.md:39]: a little hack, dose our docs builder support import content from the text file?

Comment (yaooqinn) [reply docs/deployment/high_availability_guide.md:39]: the body is foldable here, I will search a better solution later

File: docs/deployment/high_availability_guide.md [hunk 3]
```diff
@@ -15,20 +15,70 @@
...
+
+For production deployment purpose, an external zookeeper cluster is required for `kyuubi.ha.zookeeper.quorum`.
+In this mode, multiple `k.i.`s can be registered to the same ServerSpace configured by `kyuubi.ha.zookeeper.namespace` and serve together.
+
 
-Load balancing aims to optimize all Kyuubi service unit's usage, maximize throughput, minimize response time, and avoid overload of a single unit. Using multiple Kyuubi service units with load balancing instead of a single unit may increase reliability and availability through redundancy. 
+## Client-side Usage
 
-![](../imgs/ha.png)
+With [Kyuubi Hive JDBC Driver](https://mvnrepository.com/artifact/org.apache.kyuubi/kyuubi-hive-jdbc) or raw Hive JDBC Driver, a client can specify service discovery mode in JDBC connection string, i.e. `serviceDiscoveryMode=zooKeeper;` and set `zooKeeperNamespace=kyuubi;`, then it can randomly pick one of the Kyuubi service uris from the specified ZooKeeper addresses in the `/kyuubi` path.
```
Comment (pan3793) [inline docs/deployment/high_availability_guide.md:78]: "raw Hive JDBC Driver" => "vanilla Hive JDBC Driver"

**Review by pan3793 — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**